deploy/sre-prometheus/ocm-agent/100-pdb.PrometheusRule.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    prometheus: sre-pdb-managed-notification-alerts
+    role: alert-rules 
+  name: sre-pdb-managed-notification-alerts
+  namespace: openshift-monitoring
+spec:
+  groups:
+  - name: sre-pdb-managed-notification-alerts
+    rules:
+    # https://github.com/openshift/cluster-kube-controller-manager-operator/blob/3041547f5fe3227501532028e5a6993f139f6f63/manifests/0000_90_kube-controller-manager-operator_05_alerts.yaml#L35-L44
+    - alert: PodDisruptionBudgetLimit
+      expr: |
+        max by (namespace, poddisruptionbudget) (kube_poddisruptionbudget_status_current_healthy < kube_poddisruptionbudget_status_desired_healthy)
+      for: 15m
+      labels:
+        severity: Warning
+        namespace: openshift-monitoring
```

Comment:
Do we need to overwrite the namespace label here?

Reply:
i don't understand what this means. can you add more information?

Reply:
The underlying metrics will have `namespace` properties already, which is why the `expr` has `max by (namespace, poddisruptionbudget)`.  Your entry here is clobbering that label and hard-coding `openshift-monitoring`.  Christoph is wondering why you are doing that, and personally, I think dropping this line to pass along the underlying `namespace` would make it easier for the customer to find the PDB that's being complained about.

Reply:
I added https://github.com/openshift/managed-cluster-config/pull/1835/commits/ea945488f8f53fd3069948aa1baa71bd5be1d7be because otherwise CI fails. I started out without the namespace.